transition-guide.md

```diff
@@ -87,7 +87,7 @@ Now that you have your application stack on Graviton, you should run your test s
 
 **Step 6 - Performance testing**
 
-With your fully functional application its time to establish a performance baseline on Graviton. In most cases, you should expect performance gains. When comparing to existing x86-64 instances, we recommend running tests by fully loading both systems to determine the maximum possible price/performance. You can then determine and configure an appropriate load level for your production environment before performing the deployment.
+With your fully functional application its time to establish a performance baseline on Graviton. In most cases, you should expect performance gains. One of the major differences between AWS Graviton instance types and other instance types is their vCPU to physical processor core mapping. Every vCPU on a Graviton processor is a physical core. This means there is no Simultaneous Multi-Threading (SMT) and Graviton gives better linear performance scalability in most cases. When comparing to existing x86-64 instances, we recommend running tests by fully loading both systems to determine the maximum possible price/performance. You can then determine and configure an appropriate load level for your production environment before performing the deployment. If you are using Amazon EC2 Auto Scaling, consider to adjust the threshold values for the CloudWatch alarms that invoke the scaling process. This will optimise the number of EC2 instances according to the new load level that has been determined.
```

Comment:
```suggestion
With your fully functional application, it is time to establish a performance baseline on Graviton. In many cases, you can expect performance and/or capacity improvements,s depending on the workload. 

One of the major differences between AWS Graviton instance families and other Amazon EC2 instance families is their vCPU-to-physical-core mappings. By default, every vCPU on a Graviton processor maps to a physical core, and there is no Simultaneous Multi-Threading (SMT). Consequently, Graviton provides better linear performance scalability in most cases. When comparing to existing x86-64 instances, we recommend fully loading both instance types to determine the maximum sustainable load before the latency or error rate exceeds acceptable bounds. For horizontally-scalable, multi-threaded workloads that are CPU bound, you may find that the Graviton instance is able to sustain a significantly higher transaction rate before you observe unacceptable latency or error rates.

During the transition to Graviton, if you are using Amazon EC2 Auto Scaling, you may be able to increase the threshold values for the CloudWatch alarms that invoke the scaling process. This may result in a reduction in the number of EC2 instances needed to serve a given level of demand.
```